src/js/jsx/sections/libraries/LibraryBar.jsx

```diff
@@ -47,7 +47,12 @@ define(function (require, exports, module) {
          * @private
          */
         addGraphic: function () {
-            this.getFlux().actions.libraries.createGraphicFromSelectedLayer();
+            this.getFlux().actions.libraries.createGraphicFromSelectedLayer()
+                .then(function () {
+                    // Scroll to the top of the graphics list to reveal the newly added graphic.
+                    var graphicsListEle = window.document.getElementsByClassName("libraries__assets__graphic")[0];
```

Comment:
Is it the graphic _guaranteed_ to be in the DOM at this point? Just want to make sure there's no chance of dereferencing past the end of the array her.